src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java

```diff
@@ -52,7 +52,8 @@ public void copiesFromCharArrayWithCharsetToFile() throws IOException {
         final String input =
             "Hello, товарищ file #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "char array can't be copied to the file with charset UTF_8",
```

Comment:
@iiylll please use the positive form with must (e.g., "char array must be copied to the file with charset UTF_8") everywhere